docs/api/python/ndarray/random.md

```diff
@@ -51,6 +51,7 @@ In the rest of this document, we list routines provided by the `ndarray.random`
 
 .. automodule:: mxnet.random
     :members:
+    :noindex:
```

Comment:
can we also add the return values for these APIs here https://github.com/apache/incubator-mxnet/issues/13227 ? using `autodoc-process-signature` perhaps? http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html

Reply:
I think this will take a bit more research as to why `returns` and `return type` is being populated for some and not others. Since this PR fixes some broken pages, I'd rather not hold it up, and just try to tackle #13227 in another PR.